CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 ### Internal
 
+- Ask storage permission when download/share button is clicked
+- Add common download function for downloading overdue list
```

Comment:
I don't think this change made it into yesterday's release right? 😅

Reply:
Yeah, it didn't. I added this in the next release section

Reply:
Oh right, this is for the main branch. My bad 😅